homeassistant/components/openrgb/helpers.py

```diff
@@ -0,0 +1,38 @@
+"""Helper functions for the OpenRGB Integration."""
+from openrgb.utils import DeviceType
+
+from homeassistant.components.light import ENTITY_ID_FORMAT
+
+
+def orgb_tuple(color):
+    """Unpack the RGB Object provided by the client library."""
+    return (color.red, color.green, color.blue)
+
+
+def orgb_object_id(instance):
+    """Return the ORGB Devices object id."""
+    return f"{instance.name} {instance.device_id}".replace(" ", "_").lower()
```

Comment:
This is typically done like this: `return slugify(f"{instance.name} {instance.device_id}")`. This handles things like two disallowed characters in a row and characters other than just space. You'll need to import that from `homeassistant.util`